phpstan.neon

```diff
@@ -3,3 +3,7 @@ parameters:
   paths:
   - src
   - tests
+
+  ignoreErrors:
+      # Safe\DateTimeImmutable - would create overhead
```

Comment:
I would generally prefer to get instances of this class rather then its plain PHP variant. Is the overhead that significant?

Reply:
IMO it has no value here as we're not using its methods that can return false

Reply:
I think it has value when receiving the parsed date instance in a resolver. Due to the inheritance, this does not break typehints that use plain `\DateTimeImmutable`.

Reply:
It forces consumers to use that and I'm not sure it's desired. 

E.g. there's a bug in Safe in `createFromFormat()` method where it tries to call `format()` on inner value that has not been set yet so it throws NPE.

When consumer of this lib gets the Safe $instance and runs `$instance::createFromFormat()` he inherits that bug straight away. I'd rather use php's Datetime that is tested.

Reply:
Ok, sounds reasonable. You did the work, you get to decide this 😉